konig-core/src/main/java/io/konig/core/vocab/Konig.java

```diff
@@ -33,9 +33,11 @@
 	
 	public static final String DATA_SOURCE = "http://www.konig.io/ns/core/DataSource"; 
 	public static final String GOOGLE_BIG_QUERY_TABLE = "http://www.konig.io/ns/core/GoogleBigQueryTable";
+	public static final String GOOGLE_SPANNER_TABLE = "http://www.konig.io/ns/core/GoogleSpannerTable";
 	public static final String SHAPE_DATA_SOURCE = "http://www.konig.io/ns/core/shapeDataSource";
 	public static final String IRI_TEMPLATE = "http://www.konig.io/ns/core/iriTemplate";
 	public static final String BIG_QUERY_SOURCE = "http://www.konig.io/ns/core/bigQuerySource";
+	public static final String SPANNER_SOURCE = "http://www.konig.io/ns/core/spannerSource";
```

Comment:
I presume that you are simply following the pattern established by the GoogleBigQueryTable  class which declares a bigQuerySource property.  However, this property is obsolete (no longer used) and will be removed.

Let's not bring the corresponding property into the Spanner implementation.  Please remove this property and all of its references.